jekyll/_cci2/server/v4.1/air-gapped-installation/phase-2-configure-object-storage.adoc

```diff
@@ -33,39 +33,44 @@ image::./minio/minio_modify_access_policy.png[Setting circleci-data bucket acces
 
 [#copy-circleci-build-agent]
 == 2. Copy the CircleCI build agent
-Copy the latest CircleCI build agent into MinIO within your air-gapped environment.
+Follow the steps in this section to copy the pinned CircleCI build agent into MinIO within your air-gapped environment.
 
-[#download-latest-release-txt]
-=== a. Download the latest release.txt file
-Download the latest link:https://circleci-binary-releases.s3.amazonaws.com/circleci-agent/release.txt[release.txt file] from CircleCI's public S3 bucket.
+[#retrieve-pinned-agent-version]
+=== a. Extract the pinned agent version from the Helm chart
+Fetch the `circleci-server` Helm chart and extract the pinned agent version from it. Set this to the environment variable `$CIRCLE_AGENT_VERSION` so that it can be referenced in later steps.
 
 [source, bash]
 ----
-wget https://circleci-binary-releases.s3.amazonaws.com/circleci-agent/release.txt
+# Fetch the Helm chart for inspection. Replace `<version>` with the full version of CircleCI server.
```

Comment:
Including the `$` in this example makes the formatting of the snippet weird. Line numbers are automatically applied:

![Screenshot 2024-02-08 at 13 23 06](https://github.com/circleci/circleci-docs/assets/24589403/400eafc3-be8d-4a2d-b877-4ae344063db2)

Reply:
I think it would be better to remove all three `$`? What do you think?

Reply:
Yeah, I can fix that.